src/pages/case-study/central-working-changing-how-you-work.js

```diff
@@ -0,0 +1,428 @@
+    padding-bottom: ${({ theme, tablet }) =>
+      tablet && tablet.bottom ? theme.space[tablet.bottom] : null};
+  `}
+
+  ${breakpoint('desktop')`
+    padding-top: ${({ theme, desktop }) =>
+      desktop && desktop.top ? theme.space[desktop.top] : null};
+    padding-bottom: ${({ theme, desktop }) =>
+      desktop && desktop.bottom ? theme.space[desktop.bottom] : null};
+  `}
+`
+
+const IndexPage = props => {
+  const {
+    data: { contentfulNonTemplatedCaseStudyV2: caseStudy },
+    location
+  } = props
+
+  const {
+    genericBlock1: data1,
+    genericBlock2: data2,
+    genericBlock3: data3,
+    genericBlock4: data4,
+    genericBlock5: data5,
+    genericBlock6: data6,
+    genericBlock7: data7,
+    genericBlock8: data8,
+    genericBlock9: data9,
+    relatedCaseStudy
+  } = caseStudy
+
+  const outComesDataText = normalise(data2, 0).text
+  const outComesDataFigures = normaliseAll(data2.slice(1))
+
+  return (
+    <Layout location={location}>
+      <Head
+        page={{
+          ...caseStudy,
+          socialLogo: caseStudy.posterImage.file.url
+        }}
+      />
+      <CaseStudyHero caseStudy={caseStudy} />
+
+      <Grid>
+        {/* Intro  */}
+        <BlockRow
+          flexEnd
+          mobile={{ bottom: '4', top: '3' }}
+          tablet={{ bottom: '6', top: '3' }}
+        >
+          {shouldRender(data1) && (
+            <Col width={[1, 1, 1, 1, 6 / 12]}>{normalise(data1, 0).text}</Col>
+          )}
+        </BlockRow>
+      </Grid>
+      <GreyBackground>
+        <Grid>
+          {/* Outcomes */}
+          <BlockRow
+            style={{ flexDirection: 'column', alignItems: 'center' }}
+            mobile={{ bottom: '4', top: '4' }}
+            tablet={{ top: '6' }}
+          >
+            {shouldRender(data2) && (
+              <Fragment>
+                <ReactMarkdown
+                  renderers={{
+                    // eslint-disable-next-line
+                    heading: props => (
+                      <Col width={[1]} style={{ textAlign: 'center' }}>
+                        <SectionTitle {...props} />
+                      </Col>
+                    ),
+                    // eslint-disable-next-line
+                    paragraph: props => (
+                      <Col
+                        width={[1, 1, 1, 1, 9 / 12, 6 / 12]}
+                        style={{ textAlign: 'center' }}
+                      >
+                        <BodyPrimary {...props} />
+                      </Col>
+                    )
+                  }}
+                  source={outComesDataText}
+                />
+              </Fragment>
+            )}
+          </BlockRow>
+          <BlockRow
+            mobile={{ bottom: '5' }}
+            smallTablet={{ bottom: '4' }}
+            tablet={{ bottom: '6' }}
+          >
+            {outComesDataFigures.map(({ text }, index) => (
+              <Col
+                width={[1, 1, 1, 1, 4 / 12]}
+                style={{
```

Comment:
Why use inline-style here? I think it should be some kind of StyledCol to keep it consistent and use styled-component

Reply:
errr yeah I didn't want to create a specific styled component for this. Unfortunately even though Col is a `<Flex>` component the flex api hasn't been implemented which means we either  have to write inline or create a styled component